src/features/home/components/modules/VideoModule.tsx

```diff
@@ -0,0 +1,92 @@
+import colorAlpha from 'color-alpha'
+import React, { FunctionComponent } from 'react'
+import styled from 'styled-components/native'
+
+import { BlackGradient } from 'features/home/components/BlackGradient'
+import { TEXT_BACKGROUND_OPACITY } from 'features/home/components/constants'
+import { Play } from 'ui/svg/icons/Play'
+import { Spacer, Typo, getSpacing } from 'ui/theme'
+
+const THUMBNAIL_HEIGHT = 180
+
+type Props = {
+  title: string
+  videoTitle: string
+  videoThumbnail: string
+  durationInMinutes: number
+}
+
+export const VideoModule: FunctionComponent<Props> = ({
+  title,
+  videoThumbnail,
+  videoTitle,
+  durationInMinutes,
+}) => {
+  const videoDuration = `${durationInMinutes} min`
```

Comment:
Ce `durationInMinutes` il est renseigné à la main dans contentful ? On voulait pas le déduire de la durée de la vidéo youtube ?

Reply:
il me semble que le getDuration de l'api renvoie la durée de la vidéo que si elle a été load avant
de React Native Youtube-iframe : getDuration() will return 0 until the video's metadata is loaded, which normally happens just after the video starts playing.
Donc au final ca va être renseigné sur contentful à la main